ptvsd/wrapper.py

```diff
@@ -43,6 +43,7 @@
 from ptvsd.safe_repr import SafeRepr  # noqa
 from ptvsd.version import __version__  # noqa
 from ptvsd.socket import TimeoutError  # noqa
+import json
```

Comment:
Please comment all the changes in this file. Looks like linting and some tests are impacted by line and the change `on_pydevd_get_breakpoint_exception` below.